connect/cli/plugins/project/extension_helpers.py

```diff
@@ -93,9 +96,52 @@ def validate_extension_project(project_dir: str):
     extension_dict = _project_descriptor_validations(project_dir)
     _entrypoint_validations(project_dir, extension_dict)
 
+    _runner_version_validation(project_dir)
+
     click.secho(f'Extension Project {project_dir} has been successfully validated.', fg='green')
 
 
+def _runner_version_validation(project_dir):
+    res = requests.get(PYPI_EXTENSION_RUNNER_URL)
+    if res.status_code == 200:
+        latest_version = res.json()['info']['version']
+    else:
+        raise ClickException(
+            'We can not retrieve the current connect-extension-runner version from PYPI platform.'
+            '\nPlease check manually if the version on `docker-compose.yml` file is the same than the lastest on PYPI.',
+        )
+    docker_compose_file = os.path.join(project_dir, 'docker-compose.yml')
+    if not os.path.isfile(docker_compose_file):
+        raise ClickException(
+            f'There is no `docker-compose.yml` file on directory `{project_dir}`, have you deleted it?'
```

Comment:
I think here just a message saying that mandatory docker-compose.yml file is needed...rest sounds too colleague style ;)